setup.py

```diff
@@ -65,7 +65,7 @@
     "invenio-records>=1.3.0,<1.4.0",
     # upgraded packages
     "invenio-accounts-rest>=1.0.0a4,<1.1.0",
-    "invenio-app>=1.2.1,<1.3.0",
+    "invenio-app>=1.2.3,<1.3.0",
```

Comment:
This is great, but I don't see that in the config.py you override the flask static vars. I think they should be here, so that we can reproduce locally the prod env via docker-compose full

Reply:
After discussion we will not put them in the config as invenio expects them from the environment.